shell/platform/android/test/io/flutter/embedding/engine/FlutterEngineTest.java

```diff
@@ -232,4 +232,23 @@ public void itDoesNotAttachAgainWhenBuiltWithAnAttachedJNI() throws NameNotFound
 
     verify(flutterJNI, never()).attachToNative(false);
   }
+
+  @Test
+  public void itComesWithARunningDartExecutorIfJNIIsAlreadyAttached() throws NameNotFoundException {
```

Comment:
Shouldn't the test be dealing with spawned engines?

Reply:
It does. This is the "effect" of a spawned engine.

Reply:
k, looked again, sounds good